ModLinks.cs

```diff
@@ -82,9 +79,17 @@ public IEnumerable<Link> AsEnumerable()
         if (Linux is not null)
             yield return Linux;
     }
+
+    public bool Equals(Links? other)
+    {
+        if (other is null) return false;
+        return ((Windows == null && other.Windows == null) || (Windows != null && Windows.Equals(other.Windows))) && 
+            ((Mac == null && other.Mac == null) || (Mac != null && Mac.Equals(other.Mac))) && 
+            ((Linux == null && other.Linux == null) || (Linux != null && Linux.Equals(other.Linux)));
+    }
 }
 
-public class Link
+public class Link : IEquatable<Link>
```

Comment:
Similarly, I think we should just be using a record for this. Can still override Equals if you want the trim.